tests/suites/cloud/suite.js

```diff
@@ -0,0 +1,237 @@
+    const initialCommit = await this.context.get().cloud.pushReleaseToApp(this.context.get().balena.application, `${__dirname}/app`)
+    this.suite.context.set({
+      balena: {
+        initialCommit: initialCommit
+      }
+    })
+
+    // create an ssh key, so we can ssh into DUT later
+    await this.context
+      .get()
+      .cloud.balena.models.key.create(
+        this.context.get().balena.sshKey.label,
+        await this.context
+          .get()
+          .utils.createSSHKey(this.context.get().sshKeyPath)
+      );
+    this.suite.teardown.register(() => {
+      return Bluebird.resolve(
+        this.context
+          .get()
+          .cloud.removeSSHKey(this.context.get().balena.sshKey.label)
+      );
+    });
+
+    // generate a uuid
+    this.suite.context.set({
+      balena: {
+        uuid: this.context.get().cloud.balena.models.device.generateUniqueKey(),
+      },
+    });
+
+    this.suite.context.set({
+      os: new BalenaOS(
+        {
+          deviceType: this.suite.deviceType.slug,
+          network: this.suite.options.balenaOS.network,
+        },
+        this.getLogger()
+      ),
+    });
+
+    // unpack OS
+    await this.context.get().os.fetch({
+      type: this.suite.options.balenaOS.download.type,
+      version: this.suite.options.balenaOS.download.version,
+    });
+
+    // get config.json for application
+    let config = await this.context
+      .get()
+      .cloud.balena.models.os.getConfig(this.context.get().balena.application, {
+        version: this.context.get().os.contract.version,
+      });
+
+    config.uuid = this.context.get().balena.uuid;
+
+    //register the device with the application, add the api key to the config.json
+    let deviceApiKey = await this.context
+      .get()
+      .cloud.balena.models.device.register(
+        this.context.get().balena.application,
+        this.context.get().balena.uuid
+      );
+    config.deviceApiKey = deviceApiKey.api_key;
+
+    // get newly registered device id, add to config.json
+    await Bluebird.delay(1000 * 10);
+    let devId = await this.context
+      .get()
+      .cloud.balena.models.device.get(this.context.get().balena.uuid);
+    config.deviceId = devId.id;
+
+    // get ready to populate DUT image config.json with the attributes we just generated
+    this.context.get().os.addCloudConfig(config);
+
+
+    // Teardown the worker when the tests end
+    this.suite.teardown.register(() => {
+      this.log("Removing image");
+      fse.unlinkSync("/data/image"); // Delete the unpacked an modified image from the testbot cache to prevent use in the next suite
+      this.log("Worker teardown");
+      return this.context.get().worker.teardown();
+    });
+
+
+    // preload image with the single container application
+    this.log(`Device uuid should be ${this.context.get().balena.uuid}`)
+    this.log("Preloading image...");
+    console.log(`initial commit is: ${initialCommit}`)
```

Comment:
```suggestion
    this.log(`initial commit is: ${initialCommit}`)
```
What's the value in logging this?